### PR TITLE
add face detection

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

File: lib/google_ml_kit/camera_view.dart [hunk 1]
```diff
@@ -33,6 +33,7 @@ class _CameraViewState extends State<CameraView> {
   @override
   void initState() {
     super.initState();
+    print(widget.cameras.length);
```
Comment (joykuotw) [inline lib/google_ml_kit/camera_view.dart:36]: is this needed?

**Review by joykuotw — CHANGES_REQUESTED** [review]
Not able to open?
Screenshot_1630407071

File: lib/google_ml_kit/camera_view.dart (hunk 1 above)
Comment (mysticeti) [reply lib/google_ml_kit/camera_view.dart]: removed this statement in the next commit.

**Comment by mysticeti** [comment]
bug fix in the next commit. Thank you for pointing it out.

File: lib/google_ml_kit/google_ml_kit_page.dart [hunk 2]
```diff
@@ -69,23 +113,51 @@ class _GoogleMLKitPageState extends State<GoogleMLKitPage>
...
+        //       ),
+        //     ),
+        //     body: TabBarView(
+        //       controller: _tabController,
+        //       children: [
+        //         onSelectedWindow(0, localizations),
+        //         onSelectedWindow(1, localizations),
+        //       ],
+        //     ),
+        //   );
```
Comment (joykuotw) [inline lib/google_ml_kit/google_ml_kit_page.dart:134]: why leave this?

**Review by joykuotw — CHANGES_REQUESTED** [review]
not working on my side, even the bar code scanner is not working either now. You might have to show me a demo.

There is also an error while switching the camera
Screenshot_1630431266

File: lib/google_ml_kit/google_ml_kit_page.dart (hunk 2 above)
Comment (mysticeti) [reply lib/google_ml_kit/google_ml_kit_page.dart]: Removed the commented out code

**Comment by mysticeti** [comment]
The camera switching had a bug before also. I disabled the feature for now until a fix can be found.

**Review by joykuotw — APPROVED** [review]

**Pull request merged**